src/Commands/BuildCommand.php

```diff
@@ -55,8 +81,33 @@ public function handle(): void
             }
         }
 
-        $this->info((($publish ?? false) ? 'Publishing' : 'Building')." for {$os}");
+        $this->newLine();
+        intro('Updating Electron dependencies...');
+        Process::path(__DIR__.'/../../resources/js/')
+            ->env($this->getEnvironmentVariables())
+            ->forever()
+            ->run('npm update', function (string $type, string $output) {
```

Comment:
I think now that we have the NativePHP Electron plugin built in to this package, we can change this to be `npm ci` - it should mean we end up with more reproducible builds and fewer chance of spurious gotchas

Reply:
added in https://github.com/NativePHP/electron/pull/153/commits/7670441801fe240102c9af1523222e844b259edf